pages/admin.py

```
@@ -0,0 +1,3 @@
+from django.contrib import admin
```

Ruff (0.11.9)

`django.contrib.admin` imported but unused

Remove unused import: `django.contrib.admin`

(F401)